node/light.go

```diff
@@ -88,6 +95,10 @@ func openDatastore(conf config.NodeConfig, logger log.Logger) (ds.TxnDatastore,
 }
 
 func (ln *LightNode) OnStart() error {
+	if err := ln.proxyApp.Start(); err != nil {
```

Comment:
you need start in both places? `OnStart` and `newLightNode`?

Reply:
good catch, no need to start it in both places